Gemfile.lock

```diff
@@ -1047,14 +1042,13 @@ DEPENDENCIES
   sentry-sidekiq
   sidekiq (~> 6.0)
   sidekiq-scheduler (~> 5.0)
-  sidekiq_alive (~> 2.2)
   spring (~> 2.0)
   spring-watcher-listen (~> 2.0)
   sys-filesystem
   web-console (= 4.0.4)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 3.0.6p216
```

Comment:
Please don't bump Ruby -> Master is still on 0.26 we can't bump ruby to 3.0.6

Reply:
Nice catch !